### ISSUE TITLE
Manual question (typo I think)

### ISSUE THREAD

**Issue by NiltonDuarte** [ProtoRINA/users#1]
At page 9 (Pg 10 PFD)

IPC N+1 sends an M_START to IPC N+1 and after receiving it, IPC 1 replies with an M_START_R
Should be:
IPC N+1 sends an M_START to IPC 1 and after receiving it, IPC 1 replies with an M_START_R
?

**Comment by yuefeng** [comment]
Yes, you are correct. Sorry about this, and we will fix it in the next release. Thanks a lot.  Please let us know if you have any questions.
